python/lsst/cell_coadds/_single_cell_coadd.py

```diff
@@ -70,7 +70,7 @@ def __init__(
         *,
         psf: ImageD,
         inner_bbox: Box2I,
-        inputs: frozenset[ObservationIdentifiers],
+        inputs: Iterable[ObservationIdentifiers],
```

Comment:
In response to the comment in https://github.com/lsst/drp_tasks/pull/47, I've made the inputs sorted at the time of construction. This not only relieves the caller of the responsibility from sorting but also from having to convert it to a `frozenset`. It can be any iterable, and `list` or `set` are more natural ones from the caller.